src/index.html

```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8" />
     <title>Angular Getting Started</title>
-    <base href="/" />
```

Comment:
There should be a similar comment in this HTML saying that StackBlitz will try to add the `<base href="/" />` line, and that we should make sure to remove it.